### DISCUSSION TITLE
cache.app is not persisted between "deploys"

### DISCUSSION THREAD

**Discussion by flavacaster** [symfony/symfony#48319] [Q&A] [answered]
I might have been debugging this too much already so I'm not thinking straight. I've just created a new symfony project (tried 5.4 and 6.1.7) without configuring config/packages/cache.yaml and added a simple CLI command like this:

<img width="739" alt="image" src="https://user-images.githubusercontent.com/54861683/203881027-ac4b6b4b-4b2d-48ee-9979-adf56afb4b67.png">

the first time the command is executed value is calculated and saved to cache. The second time the value Is taken from cache.

After that I run cache:clear to simulate what would happen during a deploy.

Now the command is executed for the third time and there's a new value. 

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/54861683/203881617-1f74b640-242d-4cc4-92d7-d6c863a762ad.png">

While debugging I noticed that during cache:clear old files in var/cache/dev are simply deleted along with var/cache/dev/pools/app which contained original value.

Isn't cache.app supposed to persist between deploys without having to configure adapter to use some other directory ?

<img width="534" alt="image" src="https://user-images.githubusercontent.com/54861683/203881524-d307ef94-aa69-439d-b726-44be647afc58.png">

**Comment by MatTheCat** [comment, marked as answer]
I guess this is similar to https://github.com/symfony/symfony/pull/24383#issuecomment-343212210

In this case the official answer is

> You'd need to store items elsewhere.
